manifests/params.pp

```diff
@@ -37,6 +37,13 @@
   $elrepo_extras_includepkgs = absent
   $elrepo_extras_exclude = absent
 
+  #Docker CE
+  $docker_url = "https://download.docker.com/linux/centos/${::operatingsystemmajrelease}/${::architecture}/stable"
+  $docker_enabled = '1'
+  $docker_gpgcheck = '1'
+  $docker_includepkgs = absent
+  #docker_exclude = absent
```

Comment:
I don't think this should be commented out?

Reply:
huh....yaaa, not sure what happened there. Nice catch.